lib/test-case/test-case.ts

```diff
@@ -185,7 +188,7 @@ export default class TestCase {
     // create a warning file
     if (
       this.dir.hasFile('warning') &&
-      this.dir.readFile('warning') &&
+      await this.dir.readFile('warning') &&
```

Comment:
unrelated to my changes, but was giving an error-level lint. `readFile` returns a promise, which if not awaited will always be true in this context